.github/workflows/_v2_ledger_benchmark_report.yml

```diff
@@ -30,7 +30,8 @@ jobs:
       uses: dawidd6/action-download-artifact@v2
       with:
         github_token: ${{secrets.GITHUB_TOKEN}}
-        branch: feat/v2
+        #branch: feat/v2
+        branch: feat/bench-http-level # TODO: remove on merge on feat/v2
```

Comment:
Do not forget to remove that when merging !

Reply:
Yes!